client/components/ReportsPanel/ReportsTabbar.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component, PropTypes } from 'react';
+
+import './style.scss';
+
+const NavLink = props => (
+  <a
```

Comment:
Because links here should be changing sub-route (e.g. `/reports/foo` and `/reports/bar`)